libportal/portal.c

```diff
@@ -327,7 +365,7 @@ xdp_portal_init (XdpPortal *portal)
 XdpPortal *
 xdp_portal_new (void)
 {
-  return g_object_new (XDP_TYPE_PORTAL, NULL);
+  return xdp_portal_initable_new(NULL);
```

Comment:
This is an API break. It could never return `NULL` before.

I think you should move the `abort()` here. And document above:

```
If D-Bus is unavailable this API will abort. We recommend using `xdp_portal_initable_new()` to safely handle this failure.

Reply:
Also move the `g_critical()` here.

Reply:
Ok.  With this change we also no longer need a local `error` variable inside initable_new so it gets a little simpler. Code has been updated.  Thanks for the fixes and timely review.